docs/001_develop/03_client-capabilities/014_utility-methods/12_styles-util.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: 'Styles utilities'
+sidebar_label: 'Styles utilities'
+id: styles-util
+keywords: [utils, utility, styles, dynamic, look, styling, management]
+tags:
+  - utils
+  - utility
+  - styles
+  - dynamic
+  - look
+  - styling
+  - management
+---
+
```

Comment:
@patrickoneill-genesis can you advise whether this would be framework agnostic, or whether it's Genesis syntax only? If it's the latter we should put a `:::tip` here